docker-compose.yml

```diff
@@ -2,32 +2,35 @@ version: "3.8"
 
 services:
   web:
-    image: "${DHIS2_IMAGE:-dhis2/core-dev:local}"
+    image: "${DHIS2_IMAGE:-dhis2/core}"
```

Comment:
A tag  should be used here so we're not just pulling latest.

Perhaps use an environment variable, so we don't have to update the docker compose file, if we want to run a different version.